packages/dds/matrix/src/test/time/sharedMatrix.spec.ts

```diff
@@ -13,8 +13,116 @@ import {
 } from "@fluid-tools/benchmark";
 
 import { SharedMatrix } from "../../index.js";
+import { UndoRedoStackManager } from "../undoRedoStackManager.js";
 import { createLocalMatrix } from "../utils.js";
 
+/**
+ * This file contains benchmarks for measuring the execution time of operations on SharedMatrix.
+ */
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+function runBenchmark({
+	title,
+	matrixSize,
+	initialValue,
+	setup,
+	operation,
+	minBatchDurationSeconds = 0,
+	maxBenchmarkDurationSeconds,
+}: {
+	title: string;
+	matrixSize: number;
+	initialValue: string;
```

Comment:
This is the initial _cell_ value, right? Probably worth making the name more specific.